docker-compose.yml

```diff
@@ -24,27 +24,22 @@ services:
       - RELAYER_NEUTRON_CHAIN_CHAIN_PREFIX=neutron
       - RELAYER_NEUTRON_CHAIN_RPC_ADDR=tcp://node:16657
       - RELAYER_NEUTRON_CHAIN_REST_ADDR=http://node:1316
-      - RELAYER_NEUTRON_CHAIN_CHAIN_ID=test-1
       - RELAYER_NEUTRON_CHAIN_HOME_DIR=/data/test-1
       - RELAYER_NEUTRON_CHAIN_SIGN_KEY_NAME=demowallet3
       - RELAYER_NEUTRON_CHAIN_TIMEOUT=1000s
       - RELAYER_NEUTRON_CHAIN_GAS_PRICES=0.5stake
       - RELAYER_NEUTRON_CHAIN_GAS_ADJUSTMENT=1.4
       - RELAYER_NEUTRON_CHAIN_TX_BROADCAST_TYPE=BroadcastTxCommit
       - RELAYER_NEUTRON_CHAIN_CONNECTION_ID=connection-0
-      - RELAYER_NEUTRON_CHAIN_CLIENT_ID=07-tendermint-0
       - RELAYER_NEUTRON_CHAIN_DEBUG=true
       - RELAYER_NEUTRON_CHAIN_ACCOUNT_PREFIX=neutron
       - RELAYER_NEUTRON_CHAIN_KEYRING_BACKEND=test
       - RELAYER_NEUTRON_CHAIN_OUTPUT_FORMAT=json
       - RELAYER_NEUTRON_CHAIN_SIGN_MODE_STR=direct
```

Comment:
You could as well have deleted these lines (and others like in your PR in neutron-query-relayer).

I mean, if we are cleaning up, why wouldn't we clean up everything and everywhere?